src/vcpkg/base/json.cpp

```diff
@@ -366,6 +366,20 @@ namespace vcpkg::Json
         return insert(std::move(key), Value::object(value)).object();
     }
 
+    Value& Object::insert_or_replace(std::string key, std::string value)
+    {
+        auto v = get(key);
+        if (v)
+        {
+            *v = Json::Value::string(std::move(value));
+            return *v;
+        }
+        else
+        {
+            underlying_.push_back({std::move(key), Json::Value::string(std::move(value))});
+            return underlying_.back().second;
+        }
```

Comment:
```suggestion
        return insert_or_replace(std::move(key), Json::Value::string(std::move(value)));
```

This does the "two-string-moves-is-often-worse-than-copy" antipattern. But as long as it's here we may as well avoid duplicate code.